.github/workflows/template-sync-app.yml

```
@@ -0,0 +1,31 @@
+name: Template Sync with App
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)